tests/ui_tests/app/page/common/configuration/page.py

```diff
@@ -167,7 +167,17 @@ def fill_password_and_confirm_fields(self, password: str, confirmation: str, dis
     @allure.step('Click on group {title}')
     def click_on_group(self, title: str):
         """Click on group with given title"""
-        self.find_and_click(self.locators.group_btn(title))
+
+        def check_group_expanded(group: WebElement):
+            return "expanded" in group.get_attribute("class")
+
+        def click_group():
+            group = self.find_element(self.locators.group_btn(title))
+            is_expanded = check_group_expanded(group)
+            group.click()
+            assert check_group_expanded(self.find_element(self.locators.group_btn(title))) != is_expanded
```

Comment:
Add assertion message, please

and maybe make things happening here move verbose: method not only clicks on a group, but it also expects its "status" to change, so maybe add it to the method's description or smt like that

Reply:
added
waiting for "status" to change is necessary to check if the group is really opened or closed, it's just a helper